mpp/include/mpp_domains_util.inc

```diff
@@ -276,6 +276,51 @@
     return
   end subroutine mpp_get_memory_domain2D
 
+  !> @brief Modifies the indices in the domain_axis_spec type to those of the supergrid
+  subroutine mpp_set_super_grid_indices(grid)
+   type(domain_axis_spec), intent(inout) :: grid !< domain_axis_spec type
+
+   grid%begin = 2*grid%begin-1
+   grid%end = 2*grid%end+1
+   grid%size = grid%end-grid%begin+1
+
+  end subroutine mpp_set_super_grid_indices
+
+  !> @brief Modifies the indices of the input domain to create the supergrid domain
+  subroutine mpp_create_super_grid(domain)
```

Comment:
I think this name could cause confusion as people might think you are creating a supergrid from domain information.  I would suggest a name such as mpp_create_super_grid_domain.

Reply:
Renames to mpp_create_super_grid_domain in fde2caddc750a8c03c5156666e2083e6903e5369